block-buffer/Cargo.toml

```diff
@@ -14,6 +14,10 @@ readme = "README.md"
 [dependencies]
 crypto-common = "0.2.0-pre"
 generic-array = "0.14"
+zeroize = { version = "1.4", optional = true, features = ["zeroize_derive"] }
 
 [dev-dependencies]
 hex-literal = "0.3.3"
+
+[features]
+zeroize = ["dep:zeroize"]
```

Comment:
If you just remove the entire `[features]` section I'd think it would be fine.

Namespaced features aren't MSRV-compatible, as they require Rust 1.60. The MSRV being tested in CI is 1.56 (although it doesn't seem to be correct in the README.md)